compose/README.md

```diff
@@ -0,0 +1,292 @@
+# compose
+
+## Index
+
+- [Information](#information)
+- [Installation](#installation)
+- [Plan](#plan)
+- [Build](#build)
+- [Run](#run)
+  - [Local](#local)
+  - [Antithesis](#antithesis)
+- [Troubleshoot](#troubleshoot)
+- [Appendix](#appendix)
+
+## Information
+
+This document describes the process of setting up a Cardano testnet from scratch. `cardano-node` can be compiled from source or downloaded as pre-compiled binary. The testnet will run inside of **Docker Containers** and will be controlled by **Docker Compose**.
+
+## Installation
+
+Docker is the only requirement to build and run your own testnets. The following instructions are tested on Debian 12.
```

Comment:
I would remove instructions on how to install docker as 1/ these are very Debian specific and 2/ if you don't know how to install docker you should not touch this repository :)